protocol/messenger_communities.go

```diff
@@ -211,11 +211,45 @@ func (m *Messenger) handleCommunitiesHistoryArchivesSubscription(c chan *communi
 
 // handleCommunitiesSubscription handles events from communities
 func (m *Messenger) handleCommunitiesSubscription(c chan *communities.Subscription) {
-
 	var lastPublished int64
 	// We check every 5 minutes if we need to publish
 	ticker := time.NewTicker(5 * time.Minute)
 
+	recentlyPublishedOrgs := func() map[string]*communities.Community {
+		result := make(map[string]*communities.Community)
+
+		ownedOrgs, err := m.communitiesManager.Created()
+		if err != nil {
+			m.logger.Warn("failed to retrieve orgs", zap.Error(err))
+			return result
+		}
+
+		for _, org := range ownedOrgs {
+			result[org.IDString()] = org
+		}
+
+		return result
+	}()
```

Comment:
Any reason this is wrapped in an IIFE? Nothing that needs change, just curious

Reply:
It represents the intention a bit better, you can read straight away "this piece of code is responsible for obtaining recentlyPublishedOrgs". Also, this way I don't pollute the scope with unneeded `ownedOrgs` variable.